components/api/src/main/java/com/hotels/styx/api/LiveHttpResponse.java

```diff
@@ -258,10 +258,299 @@ public boolean equals(Object obj) {
+         */
+        BuilderTransformer disableCaching();
+
+        /**
+         * Makes this response chunked.
+         *
+         * @return {@code this}
+         */
+        BuilderTransformer setChunked();
+
+        /**
+         * Sets the cookies on this response by removing existing "Set-Cookie" headers and adding new ones.
+         *
+         * @param cookies cookies
+         * @return {@code this}
+         */
+        BuilderTransformer cookies(ResponseCookie... cookies);
+
+        /**
+         * Sets the cookies on this response by removing existing "Set-Cookie" headers and adding new ones.
+         *
+         * @param cookies cookies
+         * @return {@code this}
+         */
+        BuilderTransformer cookies(Collection<ResponseCookie> cookies);
+
+        /**
+         * Adds cookies into this response by adding "Set-Cookie" headers.
+         *
+         * @param cookies cookies
+         * @return {@code this}
+         */
+        BuilderTransformer addCookies(ResponseCookie... cookies);
+        /**
+         * Adds cookies into this response by adding "Set-Cookie" headers.
+         *
+         * @param cookies cookies
+         * @return {@code this}
+         */
+        BuilderTransformer addCookies(Collection<ResponseCookie> cookies);
+
+        /**
+         * Removes all cookies matching one of the supplied names by removing their "Set-Cookie" headers.
+         *
+         * @param names cookie names
+         * @return {@code this}
+         */
+        BuilderTransformer removeCookies(String... names);
+
+        /**
+         * Removes all cookies matching one of the supplied names by removing their "Set-Cookie" headers.
+         *
+         * @param names cookie names
+         * @return {@code this}
+         */
+        BuilderTransformer removeCookies(Collection<String> names);
+
+        /**
+         * Sets the (only) value for the header with the specified name.
+         * <p/>
+         * All existing values for the same header will be removed.
+         *
+         * @param name  The name of the header
+         * @param value The value of the header
+         * @return {@code this}
+         */
+        BuilderTransformer header(CharSequence name, Object value);
+
+        /**
+         * Adds a new header with the specified {@code name} and {@code value}.
+         * <p/>
+         * Will not replace any existing values for the header.
+         *
+         * @param name  The name of the header
+         * @param value The value of the header
+         * @return {@code this}
+         */
+        BuilderTransformer addHeader(CharSequence name, Object value);
+
+        /**
+         * Removes the header with the specified name.
+         *
+         * @param name The name of the header to remove
+         * @return {@code this}
+         */
+        BuilderTransformer removeHeader(CharSequence name);
+
+        /**
+         * (UNSTABLE) Removes body stream from this request.
```

Comment:
is this still applicable? I would anyway like to see a more specific description than ''stability" issues.

Reply:
No I don't think so. This is safe to use since the `ByteStream` abstraction was added.
I will remove this comment accordingly.